ethereum/scripts/dumpABI.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const args = process.argv.slice(2);
+const contract = JSON.parse(fs.readFileSync(args[0], 'utf8'));
```

Comment:
Could you not just require the json file?

Reply:
yeah I could have, though this is a once-off hacky script which will probably be discarded post-M5.